src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a) {
```

Comment:
Please change function parameter name to be more descriptive like, e.g `number` instead of `a`. Also think you should return this to keep current context `return this;`

```
add(number) {
   this.result += number;

   return this;
 },
```

```
subtract(number) {
   this.result -= number;
      
    return this;
 },
```

etc..

Please do that in all functions.